packages/rnv/src/core/schemaManager/schemaRenativeJson.js

```diff
@@ -376,6 +376,20 @@ const commonProps = {
       'If set to `true` generated js (bundle.js) files will be timestamped and named (bundle-1.0.0.js) every new version. This is useful if you want to enforce invalidate cache agains standard CDN cache policies every new version you deploy',
         examples: [true, false],
     },
+    fontSources: {
```

Comment:
I don't see fontSources being treated above. Was this added previously?

Reply:
yes. retro schema patch I noticed missing (too tiny for dedicated PR)